### DISCUSSION TITLE
Welcome to the (new) DevXan discussions

### DISCUSSION THREAD

**Discussion by Wjplatformer** [devxan/devxan#5] [General]
# :wave: Hellow. 
Please chat here. We have a funni number to leave the old discussion in. Anyways, If you want to talk to @devxan through GitHub (or if you have a question about his awesome profile README), this is a place to do it!
- :brain: Just be mindful that we are human and that everyone can see what you share here, so maybe check out the [guidelines](https://docs.github.com/articles/github-community-guidelines)
- :bulb: If you want to share an idea of change the profile README, please do so using [issues](https://github.com/devxan/devxan/issues)

**Comment by Wjplatformer** [comment]
pssp, @devxan , I'm hereeee

**Reply by devxan** [reply]
I see 👀

**Reply by Wjplatformer** [reply]
Restoration complete :)

**Comment by Wjplatformer** [comment]
@devxan , I created a new email and sent you a **verified by github** message @ totallynotxan. Let me know if you've received it. Thanks!

**Reply by devxan** [reply]
Interesting way to start emailing someone, but I guess I trust that the account is you.

**Reply by Wjplatformer** [reply]
Yeeeaaa

**Reply by devxan** [reply]
### 🤔

**Comment by Wjplatformer** [comment]
Wjplatformer.github.io/projects

**Comment by Wjplatformer** [comment]
# XAN HELP I'VE BEEN IP BANNED FROM DISCORD BECAUSE SOMEONE REPORTED THAT I WAS 2 YEARS OF AGE.
